packages/edit-site/src/components/sidebar-navigation-screen-unsupported/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Notice, __experimentalSpacer as Spacer } from '@wordpress/components';
+
+export default function SidebarNavigationScreenUnsupported() {
+	return (
+		<Spacer padding={ 3 }>
+			<Notice status="warning" isDismissible={ false }>
+				{ __(
+					'The theme you are currently using does not support current screen.'
```

Comment:
This doesn’t read right to me so here’s the smallest change I could see that would fix it:
```suggestion
					'The theme you are currently using does not support this screen.'
```
Yet, for brevity I’ll suggest:
```suggestion
					'The active theme does not support this screen.'
```